### DISCUSSION TITLE
校园网 win10专业版 v2 订阅地址均正常，但无法正常google或访问油管推

### DISCUSSION THREAD

**Discussion by Zzzdamngod** [v2fly/v2ray-core#1337] [Q&A]
各位大佬，最近去了学校换新电脑系统是win10，装了v2，导入的订阅地址在手机小火箭上都是可以跑的，以及家里用的win7用v2也是可以的。（但是新电脑的win10自己开始用也不习惯，毕竟用了太久win7。）在导入的订阅地址都是正常的情况下，或者是连了手机热点且手机热点已开小火箭手机可正常google油管这些，反而连上热点的笔记本仍是不行。后来有去下了clash同样是导入相同订阅地址一样没有反应。自己还是个小白，请问各位能帮忙出出主意嘛？
补充：尝试电脑开v2连手机热点后可正常访问，请问各位是否会是校园网问题呢？

**Comment by wangshengxin1998** [comment]
手机开启科学上网是不会影响无线热点的,也就是说手机的科学上网无法通过无线热点共享给连接的设备
PC端需要开启科学上网才行,与手机相互独立
